packages/ui/components/organization-banner/OrgBanner.test.tsx

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable playwright/missing-playwright-await */
```

Comment:
We already have a test for orgbanner. Can we not update it?

Reply:
@anikdhabal since I am creating the OrgBanner component in this PR, I'm not sure that's true.